packages/admin/src/src/i18n/en.json

```diff
@@ -125,7 +125,7 @@
   "Austria": "Austria",
   "Auth": "Auth",
   "Authentication": "Authentication",
-  "Authentication in Admin": "Authentication in Admin",
+  "Authentication in Admin": "Require authentication for the admin user",
```

Comment:
For me this sounds like it would only be required for the admin (likely you mean all users of admin but it is a bit misleading) user not for all users, what do you think?

Reply:
Hehe yes I think we have a wording overlap here ... "Admin" as adaptername was meaned by original sentence ... and was understood as "the admin user", but indeed thats not correct. Maybe "Require authentication" or "Require authentication to access Admin UI" would be alternatives

Reply:
`'Require authentication to access Admin UI' soiund best for me.
If space is limited Authentication at Admin UI" would be an alternative